doc/developing.rst

```diff
@@ -108,25 +116,79 @@ Running the documentation tests
 The documentation contains code. To check these code snippets, you
 can run this code using this command::
 
-  $ bin/sphinxpython bin/sphinx-build -b doctest doc doc/build/doctest
+  $ sphinx-build -b doctest doc doc/build/doctest
 
-If you have Make_ installed, buildout generates for you a Makefile in
-the directory ``doc/build`` that you can use::
+Since the sample code in the documentation is maintained in Python 3
+syntax, we do not support running the doctests with Python 2.7.
 
-  $ (cd doc/build; make doctest)
+Building the HTML documentation
+-------------------------------
 
-.. note::
+To build the HTML documentation (output in ``doc/build/html``), run::
 
-   Throughout this documentation, examples are using the Python 3.5 syntax.
+  $ sphinx-build doc doc/build/html
 
-.. _Make: https://en.wikipedia.org/wiki/Make_(software)
+Developing Reg, Dectate or Importscan
+-------------------------------------
 
-Building the HTML documentation
--------------------------------
+If you need to adjust the sources of Reg, Dectate or Importscan and
+test them together with Morepath, they're available in the ``src``
+directory. You can edit them and test changes in the Morepath project
+directly.
 
-To build the HTML documentation (output in ``doc/build/html``), run::
+If you want to run the tests for Morepath, Reg, Dectate and Importscan
+alltogether, you can just use::
+
+  $ py.test
+
+If you want to run the tests for one of them, let's say Reg, do::
+
+  $ cd src/reg
+  $ py.test reg
+
+Tox
+---
+
+If you want to check if Morepath works works with the supported Python versions
+and check if it passes all Travis Ci tests, you can install tox locally.
```

Comment:
Remove mention of the Travis Ci stuff here, as you haven't introduced it until the next line. There's no "locally", it's just installing tox.